CHANGELOG

```diff
@@ -1,3 +1,13 @@
+21 January 2015 - Release 0.14.4
```

Comment:
Nothing special.

I actually don't know how the process work. I've just put there that to have a place to write the highlights.

You don't do it that what? How do you do it?